perf/benchmark/README.md

```diff
@@ -162,7 +162,7 @@ Calls to Istio's Mixer component (policy and telemetry) adds latency to the side
 
     ```bash
     kubectl -n istio-system get cm istio -o yaml > /tmp/meshconfig.yaml
-    python ../../bin/update_mesh_config.py enable_mixer /tmp/meshconfig.yaml  | kubectl -n istio-system apply -
+    python ../../bin/update_mesh_config.py enable_mixer /tmp/meshconfig.yaml  | kubectl -n istio-system apply -f /tmp/meshconfig.yaml
```

Comment:
this doesn't look right, why are you piping to kubectl but then applying a file?

Reply:
Should this be `kubectl -n istio-system apply -f -`?

Reply:
> 
> 
> Should this be `kubectl -n istio-system apply -f -`?

I thought the disable_mixer was working normally, so copy that command, a closer look found that the script is also abnormal , I fix it.